components/modal/stories/modal.stories.js

```diff
@@ -3,23 +3,37 @@ import { Template } from "./template";
 
 export default {
   title: "Modal",
-  description: "The Modal component is...",
+  description: "A modal component used primarily by Dialog.",
```

Comment:
```suggestion
  description: "A modal component is a dialog box/popup window that is displayed on top of the current page."
```

Just adding a little more context.